code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -879,31 +879,27 @@
 	category = PROPERTY_TYPE_STIMULANT
 	value = 1
 
-/datum/chem_property/positive/cardiostabilizing/on_delete(mob/living/M)
-	..()
-
-	M.pain.reset_pain_reduction()
-
-/datum/chem_property/positive/cardiostabilizing/process(mob/living/M, potency = 1, delta_time)
+/datum/chem_property/positive/cardiostabilizing/process(mob/living/M, potency = 1)
```

Comment:
whilst you're at it, change M to a more detailed variable name

Reply:
💀